src/components/QueryForm.tsx

```diff
@@ -180,7 +194,50 @@ function QueryForm({
           inputValue={imagingModality}
         />
       </div>
-      <div className="row-start-10">
+      <div className="col-span-2 row-start-9">
+        <CategoricalField
+          label="Pipeline name"
+          options={Object.keys(pipelines).map((p) => ({
+            label: p.slice(3),
+            id: p,
+          }))}
+          onFieldChange={(label, value) => updateCategoricalQueryParams(label, value)}
+          inputValue={pipelineName}
+        />
+      </div>
+      {pipelineName === null ? (
+        <Tooltip
+          title={<Typography variant="body1">Please select a pipeline name</Typography>}
+          placement="right"
+        >
+          <div className="col-span-2 row-start-10">
+            <CategoricalField
+              label="Pipeline version"
+              options={[]}
+              onFieldChange={(label, value) => updateCategoricalQueryParams(label, value)}
+              inputValue={pipelineVersion}
+              disabled
+            />
+          </div>
+        </Tooltip>
+      ) : (
+        <div className="col-span-2 row-start-10">
+          <CategoricalField
+            label="Pipeline version"
+            options={[
+              { label: 'All', id: '' },
```

Comment:
minor: I don't think an "All" option is needed here. The only other place we use that is for the node-dropdowns.
Everywhere else "empty / unselected" means All. And I think that's fine.

Unrelated: does `id` not need to be set to something other than emptystring?

Reply:
> minor: I don't think an "All" option is needed here. The only other place we use that is for the node-dropdowns.
Everywhere else "empty / unselected" means All. And I think that's fine.

I believe we discussed and decided to explicitly add All for pipeline versions during stand up.

> Unrelated: does id not need to be set to something other than emptystring?

No, we filter out empty strings when generating the query url.